features/lwipstack/LWIPInterface.cpp

```diff
@@ -634,3 +634,4 @@ nsapi_error_t LWIP::Interface::bringdown()
     connected = NSAPI_STATUS_DISCONNECTED;
     return 0;
 }
+
```

Comment:
Stray editing changes in lots of files like this - please get rid of them. And squash/tidy the commits as a whole so we don't have "add bridge, remove bridge" in the history.